add id, scheme, uri, wasDerivedFrom properties (#11)
open-contracting-extensions/ocds_legalBasis_extension: README.md

Adds fields to the tender object to describe the legal basis of the contracting process – that is, the laws and regulations that govern the contracting process and that grant legal authority to the procuring entity.

> The `tender.legalBasis` object has many of the same fields as a `Classification` object including `scheme`. Example classification schemes are [LEX](https://en.wikipedia.org/wiki/Lex_(URN)), [CELEX](https://eur-lex.europa.eu/content/help/faq/intro.html#help8) and [ELI](https://en.wikipedia.org/wiki/European_Legislation_Identifier).

Reviewer: Since the extension no longer reuses the `Classification` object, I don't think we need to mention it. I think it's worth explaining what the extension now covers.

```suggestion
The `tender.legalBasis` object identifies the legal basis of the contracting process and, optionally, the law or regulation from which the legal basis was derived. Each law or regulation is identified by an `id` drawn from an identifier `scheme`. Example identifier schemes include [LEX](https://en.wikipedia.org/wiki/Lex_(URN)), [CELEX](https://eur-lex.europa.eu/content/help/faq/intro.html#help8) and [ELI](https://en.wikipedia.org/wiki/European_Legislation_Identifier).
```

> ```json

Reviewer: I think the example warrants explanation.

```suggestion
The legal basis for a contracting process is the Public Contracts Regulations 2015, which is the [transposition](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=LEGISSUM:transposition) into UK law of EU Directive 2014/24.

```json
```

## Changelog

### 2024-11-28

> * Add `wasDerivedFrom` field.

Reviewer: ```suggestion
* Add `wasDerivedFrom` object. Remove `description` field.
```
